layouts/partials/rbac-permissions-table.html

```diff
@@ -68,18 +68,18 @@ <h2 id="{{ anchorize $permission_group }}">{{ $permission_group }}</h2>
     <tbody>
       <tr>
         <td>
-          <a href="#logs_live_tail"><code>logs_live_tail</code></a>
+          <a href="/logs/guide/logs-rbac-permissions/#logs_live_tail"><code>logs_live_tail</code></a>
         </td>
         <td>Access the live tail feature</td>
         <td>false</td>
       </tr>
       <tr>
         <td>
-          <a href="#logs_live_tail"><code>logs_read_index_data</code></a>
+          <a href="/logs/guide/logs-rbac-permissions/#logs_live_tail"><code>logs_read_index_data</code></a>
```

Comment:
```suggestion
          <a href="/logs/guide/logs-rbac-permissions/#logs_read_index_data"><code>logs_read_index_data</code></a>
```

Reply:
Looks like this was previously incorrect; updating so it redirects to the correct location!